test/test_pytree.py

```diff
@@ -163,24 +163,42 @@ def test_treemap(self):
         def run_test(pytree):
             def f(x):
                 return x * 3
-            sm1 = sum(map(tree_flatten(pytree)[0], f))
+            sm1 = list(map(f, tree_flatten(pytree)[0]))
```

Comment:
Please add a comment to mention that `tree_flatten` returns flattened list and arg_spec. I was confused as to why we needed `[0]` till I saw its signature.

Reply:
Added.